solutions/observability/data-set-quality-monitoring.md

```diff
@@ -32,30 +36,51 @@ From the data set table, you’ll find information for each data set such as its
 
 Opening the details of a specific data set shows the degraded documents history, a summary for the data set, and other details that can help you determine if you need to investigate any issues.
 
-
 ## Investigate issues [observability-monitor-datasets-investigate-issues]
 
-The Data Set Quality page has a couple of different ways to help you find ignored fields and investigate issues. From the data set table, you can open the data set’s details page, and view commonly ignored fields and information about those fields. Open a logs data set in Logs Explorer or other data set types in Discover to find ignored fields in individual documents.
+The Data Set Quality page provides several ways to help you investigate issues. From the data set table, you can open the data set’s details page, open failed docs sent to the failure store in Discover, and view ignored fields.
+
+### Failure store
+
+::::{note}:
+
+**Required privileges**
+
+You need the `read_failure_store` or `all` [index privilege](../../deploy-manage/users-roles/cluster-or-deployment-auth/elasticsearch-privileges.md#privileges-list-indices) to access failure store.
+
+::::
+
+To help diagnose issues with ingestion or mapping, documents that are rejected during ingestion are sent to a dedicated data stream called failure store. From the Data Set Quality page, data streams with documents in the failure store will show a percentage in the **Failed docs (%)**. The failed docs percentage gives you a quick overview of the magnitude of potential problems in your ingestion process.
+
+Select the percentage for a specific data stream to open Discover and see the raw documents that were sent to failure store.
+
+% screenshot.
+
+To diagnose issues in a specific data stream:
+1. Select the data set name from the main table.
+1. Open **failed documents**.
 
+% screenshot
 
 ### Find ignored fields in data sets [observability-monitor-datasets-find-ignored-fields-in-data-sets]
 
 To open the details page for a data set with poor or degraded quality and view ignored fields:
 
-1. From the data set table, click ![expand icon](/solutions/images/serverless-expand.svg "") next to a data set with poor or degraded quality.
-2. From the details, scroll down to **Quality issues**.
+1. From the data set table, select a data set name.
+2. Scroll down to **Quality issues**.
 
 The **Quality issues** section shows fields that have been ignored, the number of documents that contain ignored fields, and the timestamp of last occurrence of the field being ignored.
```

Comment:
It also shows `failed documents` should we mention it here? or is it cover already with the `Failure store` section?